README.md

```diff
@@ -1,4 +1,129 @@
 # xbrl-parser
+
+[![GoDoc](https://godoc.org/github.com/polygon-io/xbrl-parser?status.svg)](https://godoc.org/github.com/polygon-io/xbrl-parser)
+
 A Go library to parse xbrl documents into their facts, contexts, and units.
 
-**Waring:** This library is under development and may change dramatically!
```

Comment:
I feel like we should probably still have this at the bottom if we haven't solidified the client interfaces.

Reply:
I'm going to cut a v1 release after this merges because I think it's stable and does enough for us to implement the financials API. Any breaking changes going forward can be a v2 release